docs/source/library/Mapping.rst

```diff
@@ -0,0 +1,28 @@
+Quantum Circuit Compilation
+===========================
+
+The main functionality for mapping quantum circuits is the :code:`compile` method. Note that this method assumes that the input quantum circuit is already decomposed into elementary gates for the specified architecture.
```

Comment:
Please convert the sentence

> Note that this method assumes that the input quantum circuit is already decomposed into elementary gates for the specified architecture.

To a `.. note::` stating

> This method assumes that the input quantum circuit is already decomposed into elementary gates for the specified architecture.

Reply:
Done.